generators/client-2/templates/src/main/webapp/app/_app.module.ts

```diff
@@ -45,7 +46,7 @@ angular
         'ngCacheBuster',
         'ngFileUpload',
         'ui.bootstrap',
-        'ui.router',
+        'ui.router.upgrade',
```

Comment:
the ng1-to-ng2 angular1 module `ui.router.upgrade` augments the angular 1 ui-router instance (`$uiRouter`) and provides ng2 capabilities such as observable parameters and ng2 "view factories"